packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -647,3 +643,16 @@ export function createOdspCacheAndTracker(
 		epochTracker,
 	};
 }
+
+export function fileEntryFromEntry(entry: IEntry, epochEntry: IFileEntry): ICacheEntry {
+	if ("file" in entry) {
+		const maybeCacheEntry = entry as ICacheEntry;
+		const mergedResolvedUrl = {
```

Comment:
i'm not sure how i feel about merging resolved urls. these are pretty complex object. did you consider adding version directly to IFileEntry like we do with docId?